tests/src/core/testqgsexpression.cpp

```diff
@@ -2056,6 +2056,15 @@ class TestQgsExpression: public QObject
 
       // get_feature_by_id
       QTest::newRow( "get_feature_by_id" ) << "get_feature_by_id('test', 1)" << true << 1;
+
+      // multi-param
+      QTest::newRow( "get_feature multi1" ) << "get_feature('test',map('col1','11','col2','test2'))" << true << 2;
+      QTest::newRow( "get_feature multi2" ) << "get_feature('test',map('col1','3','col2','test3'))" << true << 3;
```

Comment:
@roya0045 Could it be that these need to be numeric?
```suggestion
      QTest::newRow( "get_feature multi1" ) << "get_feature('test',map('col1',11,'col2','test2'))" << true << 2;
      QTest::newRow( "get_feature multi2" ) << "get_feature('test',map('col1',3,'col2','test3'))" << true << 3;
```

Reply:
usually numbers taken from a field are quoted, I can try to see if it works, debugging expressions is never fun..